_build/Makefile

```diff
@@ -2,10 +2,13 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
-PAPER         =
-BUILDDIR      = .
+SPHINXOPTS      =
+SPHINXBUILD     = sphinx-build
+PAPER           =
+BUILDDIR        = .
+SPHINXAUTOBUILD = sphinx-autobuild
+IGNOREWATCH		= --ignore "/www/_build" \
+                  --ignore "/www/.idea"
```

Comment:
Sounds arbitrary. what's about .git, `~` files, and other"ignored" file. 
Does it make sens to use a whitelist instead of a blacklist?
What's about generating the list based on .gitignore files?

Reply:
It is arbitrary indeed. But this is how the lib works (see [their doc](https://pypi.org/project/sphinx-autobuild/)). Because of the file structure (_build is part of the root folder for the documentation and documentation is entirely on root too) there is not much I see that I could do.

Parsing .gitignore to add the files to this list seems overcomplicated to me.

Reply:
This is just relevant for while developing, the generated content is not what is shipped, the source code is. Generating it on the fly has the objective of helping the dev to spot and fix the issues quicker instead of waiting for the entire re-build.

Having said that. I remove all the arbitrary ignore and just left the folder that contains the builder content to prevent circular references.

Hope this is enough.